src/utility/VersionUtility.ts

```diff
@@ -48,28 +45,14 @@ export class VersionUtility {
     }
 
     // Returns the CLI version that this VSCode session expects
-    public static async getExpectedCliVersionAsync(context: vscode.ExtensionContext, experimentationService: IExperimentationService, packageJsonContent: object, logger: Logger): Promise<string> {
-        const bridgeEnvironment = EnvironmentUtility.getBridgeEnvironment(logger);
-
-        // This call will return the upcoming version for treatment group; control & everyone else will get the stable prod version. Undefined if the flight is currently inactive.
-        // TODO(ansoedal): Disable ExP permanently. For now this call will return undefined, as long as the treatment variable doesn't exist
-        const expBinariesVersion = experimentationService.getTreatmentVariable<string>(/*configId*/ `vscode`, /*name*/ `mindaroBinariesVersion-doesnotexist`);
-        logger.trace(`Call to ExP returned version '${expBinariesVersion}'`);
-
+    public static async getExpectedCliVersionAsync(packageJsonContent: object): Promise<string> {
         // How we resolve the expected CLI version:
         //  1. Is the BRIDGE_CLI_VERSION environment variable set? Use that one.
-        //  2. Is a flight in progress? Get the binaries version from ExP.
-        //  3. Fall back onto whichever LKS version was specified in package.json. (For dev and staging, this is whichever version matches this extension. For Prod, this is a variable we specify in the Mindaro-Connect-VSCode pipeline.)
+        //  2. If BRIDGE_CLI_VERSION is not set, use the version specified in package_Json (irrespective of the env - dev/prod)
+        //  3. If bridge build path is not set to null, then expected CLI version will be null
         const packageJsonVersion: string = packageJsonContent[`extensionMetadata`][`expectedCLIVersion`];
         const expectedCLIVersion = process.env.BRIDGE_CLI_VERSION != null ? process.env.BRIDGE_CLI_VERSION :
-            (expBinariesVersion != null && bridgeEnvironment === Environment.Production ? expBinariesVersion :
-            (process.env.BRIDGE_BUILD_PATH != null ? null :
-            (packageJsonVersion != null && !packageJsonVersion.startsWith(`$`) ? packageJsonVersion : null)));
-
-        logger.trace(`Resolved expected CLI version '${expectedCLIVersion}'`);
-        if (expectedCLIVersion != null && !/^\d+(\.\d+){3}$/.test(expectedCLIVersion)) {
-            throw new Error(`Invalid minimum CLI version '${expectedCLIVersion}' found in package.json file.`);
-        }
+            (process.env.BRIDGE_BUILD_PATH === null ? packageJsonVersion : null);
```

Comment:
```suggestion
            (process.env.BRIDGE_BUILD_PATH == null ? packageJsonVersion : null);
```

Reply:
Hi when env var BRIDGE_BUILD_PATH is not specified, `process.env.BRIDGE_BUILD_PATH` returns `undefined` but not `null` so `process.env.BRIDGE_BUILD_PATH === null` returns false, this causes `expectedCLIVersion = null` even when BRIDGE_BUILD_PATH is not specified.